editors/vscode/src/common.ts

```diff
@@ -85,12 +91,23 @@ export class PropertiesViewProvider implements vscode.WebviewViewProvider {
                             response: response,
                         });
                     });
+                    break;
+                case "remove_binding":
+                    remove_binding(
+                        data.document,
+                        data.element_range,
+                        data.property_name,
+                    ).catch((_) => {
+                        // catch this to avoid errors showing up in the console
```

Comment:
Hm, if we're sending a command to our LSP, don't we want to know somehow when something goes wrong - for diagnostic purposes? Like via a console.log at least, not a popup?